src/frontend/sharedComponents/LeaveProjectModalContent/index.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {useAcceptInvite} from '../../hooks/server/invites';
+import {AlreadyOnProject} from './AlreadyOnProject';
+import {LeaveProject} from './LeaveProject';
+import {useProjectSettings} from '../../hooks/server/projects';
+import {LeaveProjectModalState} from '../ProjectInviteBottomSheet';
+
+type LeaveProjectModalContentProps = {
+  closeSheet: () => void;
+  inviteId: string;
+  accept: ReturnType<typeof useAcceptInvite>;
+  leaveModalState: LeaveProjectModalState;
+  setToLeaveProject: () => void;
+};
+
+export const LeaveProjectModalContent = ({
+  closeSheet,
+  inviteId,
+  accept,
+  leaveModalState,
+  setToLeaveProject,
+}: LeaveProjectModalContentProps) => {
+  const {data} = useProjectSettings();
```

Comment:
would it make sense to move this to `LeaveProject`?

Reply:
whoops, both `AlreadyOnProject` and `LeaveProject` can take a name prop, thats why its at the parent level. Good catch

Reply:
https://github.com/digidem/comapeo-mobile/pull/526/commits/bb42f8e69c0b9614d8bd1cc8b2d69fd71c59a832